ftl/common/ftl_util.py

```diff
@@ -158,3 +158,55 @@ def generate_overrides(set_path):
         'env': env
     }
     return overrides_dct
+
+
+def parseCacheLogEntry(entry):
```

Comment:
nit:  it might be easier to maintain the parser if we used a shared template for the parsing and the cache entry itself.  there will be extra effort currently to add new information to both the Cache format and then update the parser as it stands.  It also might make the parser easier to read/understand

Reply:
yeah agreed. if we start making a lot of changes to the cache entry format then I can change it over to using a template, but I think it should remain relatively unchanged so it shouldn't be too much work.